svc/FetchService.js

```diff
@@ -25,6 +29,18 @@ import {stringify} from 'qs';
 export class FetchService {
 
     autoAbortControllers = {};
+    appHeaders = {};
+
+    /**
+     * Set App-wide headers for all subsequent requests. App-wide headers are not
+     * sent with core framework requests.
+     *
+     * @param {Object} headers - Headers to be sent with all requests.
+     */
+    setAppHeaders(headers) {
+        throwIf(!isPlainObject(headers), 'Todo');
```

Comment:
I think we can leave this off - again it would be called once, presumably by a dev who has read up on or been informed of this feature and its requirements as documented above.

Reply:
agree